### PR TITLE
Release v0.61.0

### LINKED ISSUE #1130: Release the operator v0.61.0

### PR DESCRIPTION
**Pull request by yuriolisa**
Signed-off-by: Yuri Sa <yurimsa@gmail.com>

Resolves #1130 

Blocked by 
- [x] https://github.com/open-telemetry/opentelemetry-operator/pull/1116
- [x] https://github.com/open-telemetry/opentelemetry-operator/pull/1126
- [x] https://github.com/open-telemetry/opentelemetry-operator/issues/1143

### CODE REVIEW

**Comment by yuriolisa** [comment]
@jpkrohling, @pavolloffay, @VineethReddy02.

**Comment by pavolloffay** [comment]
This is blocked by #1116 

cc) @frzifus

**Comment by frzifus** [comment]
yes, iam working on it since this afternoon. Drives me crazy. i will keep you updated.

**Comment by pavolloffay** [comment]
@yuriolisa please rebased the PR, the CI config has changed a bit.

**Comment by yuriolisa** [comment]
@pavolloffay, @jpkrohling. could you please have a look?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 Changes by Version
 ==================
+0.61.0
+-------------------
+### 🚀 New components 🚀
+* [HPA] Add targetCPUUtilization field to collector config ([#1066](https://github.com/open-telemetry/opentelemetry-operator/pull/1066), [@moh-osman3](https://github.com/moh-osman3))
+### 💡 Enhancements 💡
+* Add default settings to README ([#1114](https://github.com/open-telemetry/opentelemetry-operator/pull/1114), [@timbuchwaldt](https://github.com/timbuchwaldt))
+
```
Comment (frzifus) [inline CHANGELOG.md:9]:
```Suggestion

#### :x: Breaking Changes :x:
- Jaeger receiver no longer supports remote sampling. To be able to perform an update, it must be deactivated or replaced by a configuration of the [jaegerremotesampling](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.61.0/extension/jaegerremotesampling) extension. ([otel-contrib#14707](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/14707)

```

Depends on: https://github.com/open-telemetry/opentelemetry-operator/pull/1116

**Review by frzifus — CHANGES_REQUESTED** [review]

File: versions.txt [hunk 2]
```diff
@@ -2,10 +2,10 @@
...
 # the docs as well.
-opentelemetry-collector=0.60.0
+opentelemetry-collector=0.61.0
 
 # Represents the current release of the OpenTelemetry Operator.
-operator=0.60.0
+operator=0.61.0
 
 # Represents the current release of the Target Allocator.
 targetallocator=0.1.0
```
Comment (jaronoff97) [inline versions.txt:11]: @yuriolisa can you also bump the target allocator version please?

```suggestion
targetallocator=0.61.0
```

**Review by jaronoff97 — CHANGES_REQUESTED** [review]

**Comment by bryan-aguilar** [comment]
Any updates/eta on the release or the blocking PRs?

**Comment by frzifus** [comment]
> Any updates/eta on the release or the blocking PRs?

Just from my side - Ive made hopefully a final adjustments to the jaeger upgrade routine. Now waiting for a review.

**Comment by pavolloffay** [comment]
@yuriolisa could you please rebase the PR and update changelog - there have been some features merged into the main branch recently. 

@frzifus could you please work with @yuriolisa to document the breaking change of Jaeger the receiver in the changelog?

**Review by frzifus — COMMENTED** [review]
added all changes [here](https://github.com/open-telemetry/opentelemetry-operator/pull/1131/files#r987937895).

**Comment by Kausik-A** [comment]
Any updates/eta on the release?

File: versions.txt (hunk 2 above)
Comment (VineethReddy02) [reply versions.txt:11]: We do not bump the target allocator version in line with operator releases. It's always the `0.1.0` as we treat the target allocator as a subsystem with its own versioning. 

If there are any major improvements/breaking changes we can bump it with minor/patches release versions.

Comment (pavolloffay) [reply versions.txt:11]: That has changed https://github.com/open-telemetry/opentelemetry-operator/blob/main/RELEASE.md.

@jaronoff97 maybe there should be a single version for operator and TA. The TA image could be released if that version changes.

**Comment by ekarlso** [comment]
Can we get this in ?

**Comment by yuriolisa** [comment]
Sorry for the late response. I've just rebased the PR. @pavolloffay, could you please review it?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,16 @@
 Changes by Version
 ==================
+0.61.0
+-------------------
+### 🚀 New components 🚀
+* [HPA] Add targetCPUUtilization field to collector config ([#1066](https://github.com/open-telemetry/opentelemetry-operator/pull/1066), [@moh-osman3](https://github.com/moh-osman3))
+### 💡 Enhancements 💡
+* Add default settings to README ([#1114](https://github.com/open-telemetry/opentelemetry-operator/pull/1114), [@timbuchwaldt](https://github.com/timbuchwaldt))
```
Comment (pavolloffay) [inline CHANGELOG.md:8]: This does not have to be here, it's just docs.

**Review by pavolloffay — COMMENTED** [review]
@yuriolisa could you please update the changelog? There have been various things merged to the main that are worth mentioning.

**Comment by yuriolisa** [comment]
@frzifus, could you please review the Changelog?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,35 @@
 Changes by Version
 ==================
+0.61.0
+-------------------
+### 🚀 New components 🚀
+* [HPA] Add targetCPUUtilization field to collector config ([#1066](https://github.com/open-telemetry/opentelemetry-operator/pull/1066), [@moh-osman3](https://github.com/moh-osman3))
+* Add ingress options ([#1128](https://github.com/open-telemetry/opentelemetry-operator/pull/1128), [@frzifus](https://github.com/frzifus))
```
Comment (frzifus) [inline CHANGELOG.md:7]:
```suggestion
* extend otelcol crd with minimalistic ingress options ([#1128](https://github.com/open-telemetry/opentelemetry-operator/pull/1128), [@frzifus](https://github.com/frzifus))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,35 @@
...
+* Add missing nil checks in collector validating webhook ([#1136](https://github.com/open-telemetry/opentelemetry-operator/pull/1136), [@kristinapathak](https://github.com/kristinapathak))
+* Fix lint issues in target allocator ([#1090](https://github.com/open-telemetry/opentelemetry-operator/pull/1090), [@kristinapathak](https://github.com/kristinapathak))
+### 🧰 Bug fixes 🧰
+* Fix generated deepcopy file changes check ([#1154](https://github.com/open-telemetry/opentelemetry-operator/pull/1154), [@pavolloffay](https://github.com/pavolloffay))
+* Fix Target Allocator builds by using versions.txt ([#1140](https://github.com/open-telemetry/opentelemetry-operator/pull/1140), [@jaronoff97](https://github.com/jaronoff97))
+* Add missing entry to 0.60.0 changelog ([#1102](https://github.com/open-telemetry/opentelemetry-operator/pull/1102), [@pavolloffay](https://github.com/pavolloffay))
+#### OpenTelemetry Collector and Contrib
+* [OpenTelemetry Collector - v0.61.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.61.0)
+* [OpenTelemetry Contrib - v0.61.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.61.0)
+
```
Comment (frzifus) [inline CHANGELOG.md:32]:
```suggestion

#### :x: Breaking Changes :x:
- Jaeger receiver no longer supports remote sampling. To be able to perform an update, it must be deactivated or replaced by a configuration of the [jaegerremotesampling](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.61.0/extension/jaegerremotesampling) extension. To increase the collector version afterwards, the update must be triggered again by restarting the operator. Alternatively, the `OpenTelemetryCollector` CRD can be re-created. ([otel-contrib#14707](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/14707)

```

**Review by frzifus — CHANGES_REQUESTED** [review]
Almost good with it. Seems the Breaking change warning got dropped.

**Review by frzifus — APPROVED** [review]
lgtm, thanks @yuriolisa :)

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,37 @@
...
+* Change error logs to info for building container ([#1146](https://github.com/open-telemetry/opentelemetry-operator/pull/1146), [@kristinapathak](https://github.com/kristinapathak))
+* Add missing nil checks in collector validating webhook ([#1136](https://github.com/open-telemetry/opentelemetry-operator/pull/1136), [@kristinapathak](https://github.com/kristinapathak))
+* Fix lint issues in target allocator ([#1090](https://github.com/open-telemetry/opentelemetry-operator/pull/1090), [@kristinapathak](https://github.com/kristinapathak))
+### 🧰 Bug fixes 🧰
+* Fix generated deepcopy file changes check ([#1154](https://github.com/open-telemetry/opentelemetry-operator/pull/1154), [@pavolloffay](https://github.com/pavolloffay))
+* Fix Target Allocator builds by using versions.txt ([#1140](https://github.com/open-telemetry/opentelemetry-operator/pull/1140), [@jaronoff97](https://github.com/jaronoff97))
+* Add missing entry to 0.60.0 changelog ([#1102](https://github.com/open-telemetry/opentelemetry-operator/pull/1102), [@pavolloffay](https://github.com/pavolloffay))
+#### :x: Breaking Changes :x:
```
Comment (pavolloffay) [inline CHANGELOG.md:29]: Please move this on top to make it visible. First should be breaking changes, deprecations and then other changes.

**Pull request merged**